arduino-cli.py

```diff
@@ -0,0 +1,127 @@
+# encoding: utf-8
+
+import argparse
+import json
+import sys
+
+from workflow import Workflow, ICON_WEB, web
+
+log = None
+
+
+def run_command(cmd):
+    from invoke import run
+    
+    cmd = "./arduino-cli {cmd} --format json".format(cmd=cmd)
+
+    result = run(cmd, hide=True, warn=True)
+    if not result.ok:
+        raise Exception(result.stdout)
+
+    return json.loads(result.stdout)
+
+
+def search_key_for_board(board):
+    elements = []
+    elements.append(board["name"])
+    elements.append(board["FQBN"])
+    return u" ".join(elements)
```

Comment:
No need for u

Reply:
Honestly, I don't remember why I added them, but are non required indeed.. thanks!